src/main/kotlin/com/th/plu/domain/answer/Answer.kt

```diff
@@ -0,0 +1,40 @@
+package com.th.plu.domain.answer
+
+import com.th.plu.domain.common.BaseEntity
+import com.th.plu.domain.member.Member
+import com.th.plu.domain.question.Question
+import jakarta.persistence.*
+import lombok.AccessLevel
+import lombok.AllArgsConstructor
+import lombok.Builder
+import lombok.NoArgsConstructor
+import java.time.LocalDateTime
+
+
+@Table(name = "answers")
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Builder(access = AccessLevel.PRIVATE)
+class Answer(
+
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "answer_id")
+    private var id: Long? = null,
```

Comment:
```suggestion
    val id: Long? = null
```

pk 값같은 경우는 캡슐화 + reflection 으로 var 보다는 val

Reply:
> pk 값같은 경우는 캡슐화 + reflection 으로 var 보다는 val

오호 하나 배워갑니다~!

Reply:
> pk 값같은 경우는 캡슐화 + reflection 으로 var 보다는 val

private도 지워야하나요??

Reply:
private 안지우면 외부에서 getter 접근이안됩니다

Reply:
> private 안지우면 외부에서 getter 접근이안됩니다

kotlin에서 val은 getter 메서드를 만들어준다고 생각했는데, 알아보니 그게 아니라 필드(프로퍼티)에 직접 접근해서 값을 얻어오는 것이었네요! 하나 배워갑니다 감사합니다:)